### ISSUE TITLE
B1 presets

### ISSUE THREAD

**Issue by drronnie85** [xoseperez/espurna#1059]
Does B1 suppot presets and way to change it by switch? I want create two presets: bright for day and dim for night, and change it quick by clicking switch. For example ON OFF ON - first preset, ON OFF ON OFF ON - second?
Thank you

**Comment by xoseperez** [comment]
No, that's not implemented. Maybe someone would like to contribute?
